test/testinput/state.yml

```diff
@@ -1,11 +1,25 @@
 geometry:
-  # insert other model specific things here
+  grid: S360x180
+  domain:
+    type: global
+    west: -180
+
+read_from_file: 1
 
 state test:
-  norm file: 0.0
+  norm file: 17904.548097851199  # 300.0
   tolerance: 1.0e-12
-  date: &date 2018-04-15T00:00:00Z
+  date: &date 1985-01-01T12:00:00Z
+  read_from_file: 1
   statefile:
+    read_from_file: 1
     date: *date
-    state variables: &state_vars [statevar1, statevar2]
-    # insert other model specific things here
\ No newline at end of file
+    filename: /glade/scratch/lgchen/data/SST_ML/ESA_SST_CCI/L4_Analysis_CDRv2.1_Regrided_1x1/nc_180/19850101_regridded_sst.nc
+    state variables: &state_vars [sst]
```

Comment:
I would recommend renaming the state variable from `sst` to `sea_surface_temperature`. I realize the input file is probably using `sst`, but once inside JEDI we try to use the CF naming conventions whenever possible (http://cfconventions.org/Data/cf-standard-names/76/build/cf-standard-name-table.html). I also realize it doesn't make much of a difference now, but might be helpful further along when hooking up to UFO

Reply:
@travissluka I will rename sst to sea_surface_temperature at the next step.